website/content/docs/concepts/domain-model/credential-stores.mdx

```diff
@@ -58,6 +58,10 @@ A Vault credential store has the following additional attributes:
 - `namespace` - (optional)
   A Vault [namespace][]. Requires Vault Enterprise.
 
+### Static Credential Store Attributes
+
+A Static credential store has no additional attributes and only supports the optional attributes.
```

Comment:
This line reads a bit confusing to me. Generally if there are optional type specific attributes it is part of the `attributes` map so I think this should be more like "only supports resource parameters" or some such thing. That suggests also changing it above where there is the Attributes header to something more standardized.

I'm not sure how widespread this pattern is right now but probably we want to draw a distinction between things that go in the `attributes` map and thus are truly attributes and things that are common across resources.

Reply:
can you suggest a fix? @irenarindos is the original author.

Reply:
"A static credential store has no type-specific attributes." is my suggestion.